### ISSUE TITLE
Why was the training of UniAD without validation

### ISSUE THREAD

**Issue by rujiao** [OpenDriveLab/UniAD#165]
Facts:
1) According to the paper, the performance of UniAD in comparison to prior state-of-the-arts was tested on the nuScenes validation
set.
2) According to the training log in release 1.0 and 1.1, both test set and validation set are set as nuScenes validation
set. The validation ran only once in the end after 20 training epochs.
Questions:
Why was the training of UniAD without validation? 
How do you know that training for 20 epochs is not involved in a early stopping or overfitting?  
Thank you!

**Comment by Rango-T10000** [comment]
I have the same question since that workflow = {train : 1} in this code.
